### ISSUE TITLE
try/except lambda lifter error

### ISSUE THREAD

**Issue by ddjoh** [actonlang/acton#966]
### Acton Version

0.11.7.20221012.13.41.22

### Steps to Reproduce

Compile:
```
def test():
    try:
        raise ValueError("")
    except ValueError:
        pass
```

### Expected Behavior

Success

### Actual Behavior

```
Building project in /home/ddjoh/workspaces/act-test-1
  Compiling test.act for release
actonc: ll unexpected: try:
    $RAISE(ValueError(""))
except ValueError:
    pass
CallStack (from HasCallStack):
  error, called at ././Acton/LambdaLifter.hs:219:43 in main:Acton.LambdaLifter
```

**Comment by plajjan** [comment]
@nordlander is this issue valid on its own or is this merely a symptom of "exceptions are not implemented completely" ? i.e. should we meld this into a larger issue "to implement exceptions"?

**Comment by plajjan** [comment]
Fixed by #1463. Closing.
